backend/src/controllers/Auth.js

```diff
@@ -35,49 +27,38 @@ module.exports = {
                 isaf_status: false,
                 verify_status: false,
                 picture: user.picture,
-            })
+            });
 
             try {
-                const new_student_db = await Student.create(new_student)
-                const fetch_student = await Student.findOne({ email: user.email })
-                const studentWithKey = {
-                    id: fetch_student.id,
-                    name: fetch_student.name,
-                    email: fetch_student.email,
-                    isaf_status: fetch_student.isaf_status,
-                    verify_status: fetch_student.verify_status,
-                    picture: fetch_student.picture,
-                    key: jwt.sign(
-                        {
-                            email: fetch_student.email,
-                        },
-                        process.env.JWT_SECRET,
-                        { expiresIn: '30d' }
-                    ),
-                }
-
-                return res.status(200).json(studentWithKey) // return the new user with the jwt key valid for 30 days
+                await Student.create(new_student);
+                const fetch_student = await Student.findOne({ email: user.email });
+                let { ...payload } = fetch_student;
```

Comment:
Wondering how this part of the code works, would 
`const payload = await Student.findOne({ email: user.email });`
be the same since we are destructuring and reassigning anyways.

Reply:
`user.email` is extracted from the jwt key that the backend receives in the header for endpoint protection. If the jwt key is signed by us then we can use the information in it to search our student database to look for the user and fetch the information that will be sent back to the app in the response.

Reply:
Ohh I see, I was more specifically just wondering about line 34/35